### ISSUE TITLE
Panels should be more eager to collapse on window resize

### ISSUE THREAD

**Issue by robintown** [vector-im/element-web#17261]
It's always been a bit annoying that the right panel refuses to auto-collapse no matter how small you make the window, but especially now that there's another expandable panel on screen (the space panel), the timeline can get squished into oblivion quite easily whenever I try to have two windows side-by-side on my laptop screen. It would be nice to not have to manually fiddle with panels when I just need the window to be a bit smaller.

Also, I think it would be nice to have the room list collapse *last* as it tends to be the most important panel to see, though I don't know if others agree.

**Comment by SimonBrandner** [comment]
> Also, I think it would be nice to have the room list collapse last as it tends to be the most important panel to see, though I don't know if others agree.

Yeah, the space panel should be eventually resizable, so its behaviour will change, so that might get fixed
